README.md

```diff
@@ -114,6 +114,48 @@ To run `DefaultAzureCredentailAdapterTest`, ensure you have `.env` file created
 
 Once you have the `.env` file configured, run the test using JUnit 5 runner.
 
+## TypeScript
+
+### DefaultAzureCredentialAdapter.ts
```

Comment:
Please rename to AzureIdentityCredentialAdapter and update all associated files and content to match that new name.  Thanks.